### ISSUE TITLE
usdview no longer runs without --defaultsettings

### ISSUE THREAD

**Issue by jcowles** [PixarAnimationStudios/USD#809]
### Description of Issue
When I run usdview, I get the following:

```
Traceback (most recent call last):
  File "C:\src\usd\_build-monolithic-py\lib\python\pxr\Usdviewq\stageView.py", line 2131, in glDraw
    if not self._getRenderer():
  File "C:\src\usd\_build-monolithic-py\lib\python\pxr\Usdviewq\stageView.py", line 907, in _getRenderer
    self._handleRendererChanged(self.GetCurrentRendererId())
  File "C:\src\usd\_build-monolithic-py\lib\python\pxr\Usdviewq\stageView.py", line 914, in _handleRendererChanged
    self._rendererDisplayName = self.GetRendererDisplayName(rendererId)
  File "C:\src\usd\_build-monolithic-py\lib\python\pxr\Usdviewq\stageView.py", line 932, in GetRendererDisplayName
    return self._renderer.GetRendererDisplayName(plugId)
pxr.Tf.ErrorException:
        Error in 'pxrInternal_v0_19__pxrReserved__::UsdImagingGLEngine::GetRendererDisplayName' at line 654 in file C:\src\usd\USD\pxr\usdImaging\lib\usdImagingGL\engine.cpp : 'Failed verification: ' HdxRendererPluginRegistry::GetInstance(). GetPluginDesc(id, &pluginDescriptor) ''
```

When I run with "--clearsettings", I get this message:

`INFO: ClearSettings requested, but there were no settings currently stored.`

Running with "--defaultsettings" fixes the problem, but I must ALWAYS use this option now.

Here are my state and state.json files (zipped):
[state.zip](https://github.com/PixarAnimationStudios/USD/files/3020885/state.zip)

### Steps to Reproduce
Unknown.

### System Information (OS, Hardware)
Windows

### Package Versions
19.01

### Build Flags
--build-monolithic --openimageio --alembic

**Comment by c64kernal** [comment]
Hey @jcowles -- is this specific to 19.01?  We haven't been able to reproduce this with dev so far...

**Comment by jilliene** [comment]
Filed as internal issue #USD-5176

**Comment by jcowles** [comment]
@c64kernal I've only tried it on 19.01, so maybe? It also seems like something has gotten into a strange state, so I wouldn't be surprised if it's hard to repro.

**Comment by c64kernal** [comment]
Sounds good, thanks @jcowles -- we'll keep an eye out!

**Comment by superfunc** [comment]
@jcowles it may be worth trying to nuke the state file and see if that resets it.

**Comment by jcowles** [comment]
@superfunc yes, that fixes the problem (which I expected)

@c64kernal no, this is not happening on 19.05 with the old .usdview state files

Closing, since this seems to be fixed in 19.05
